lib/collection/src/collection_manager/optimizers/segment_optimizer.rs

```diff
@@ -44,6 +44,9 @@ pub struct OptimizerThresholds {
 /// The selection of the candidates for optimization and the configuration
 /// of resulting segment are up to concrete implementations.
 pub trait SegmentOptimizer {
+    /// Get name describing this optimizer
+    fn name(&self) -> &str;
```

Comment:
It seems all implementors are returning `x optimizer`.
Maybe they could just return `x` as they are all optimizers to avoid repetition.

Reply:
Yeah, couldn't decide on this myself. Will do!